src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bf_tiled.cl

```diff
@@ -85,8 +85,373 @@
 #   define INPUT_ELEMENTS_COUNT INPUT0_ELEMENTS_COUNT
 #endif
 
-REQD_SUB_GROUP_SIZE(SIMD)
-KERNEL(fc)(
+inline void FUNC(fc_bf_tiled_kernel_tile_b1)(
+    OPTIONAL_SHAPE_INFO_ARG
+    const __global INPUT0_TYPE* input,
+#if DECOMPRESSION_SCALE_TERM
+    const __global DECOMPRESSION_SCALE_TYPE* decompression_scale,
+#endif
+#if DECOMPRESSION_ZP_TERM && !DECOMPRESSION_ZP_SCALAR
+    const __global DECOMPRESSION_ZP_TYPE* decompression_zp,
+#endif
+    __global OUTPUT_TYPE* output,
+    const __global FILTER_TYPE* weights
+#if BIAS_TERM
+    , const __global BIAS_TYPE* biases
+#endif
+#if HAS_FUSED_OPS_DECLS
+    , FUSED_OPS_DECLS
+#endif
+) {
+    #define FORCED_TILE_B 1
```

Comment:
Why not just using BATCH_SIZE ? This function is used only when BATCH_SIZE == 1 so seems to be same.

Reply:
My intention was to explicitly specify that used TILE_B size in this implementation is always 1
As for BATCH_SIZE, this implementation is targeted only to support dynamic cases, and in that case it may contain shape_info[] references as well